### PR TITLE
chore(tensorrt_yolox): rework parameters

### PR DESCRIPTION
**Pull request by tzhong518**
## Description

Use config for parameters

## Tests performed

Launch tensorrt_yolox and confirm that it loads parameters successfully

## Effects on system behavior

Not effect.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by kminoda — DISMISSED** [review]
LGTM

File: perception/tensorrt_yolox/config/yolox_s_plus_opt.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**:
```
Comment (kminoda) [inline perception/tensorrt_yolox/config/yolox_s_plus_opt.yaml:1]: Please rename the file name to param.yaml

Comment (tzhong518) [reply perception/tensorrt_yolox/config/yolox_s_plus_opt.yaml]: fixed in https://github.com/autowarefoundation/autoware.universe/pull/6239/commits/d88251c03f73f4c597850b6225d5e0efc23db8a3

**Review by kminoda — APPROVED** [review]

**Review by miursh — APPROVED** [review]

**Comment by kminoda** [comment]
@manato Hi, would you also check this PR?

**Comment by manato** [comment]
@kminoda Thank you very much for your update! The modification looks good to me roughly.
One quick question: By introducing this modification, the descriptions of each variable look disappear. I guess these descriptions are useful for users who would like to tune the variables. Do you think you can leave them as comments in the yaml? I'd appreciate it if you ignore the proposal in case that it violates some agreements,

**Comment by kminoda** [comment]
@tzhong518 Friendly ping

**Comment by tzhong518** [comment]
> @tzhong518 Friendly ping

added in https://github.com/autowarefoundation/autoware.universe/pull/6239/commits/f6cae459f85abafb96401ec40975c5b7b13f0719

**Comment by kminoda** [comment]
@manato Hi, would you check this again?

**Review by manato — APPROVED** [review]
LGTM. Thanks for your contribution!

**Pull request merged**